README.md

```diff
@@ -1 +1,360 @@
+https://service.centrapay.com/payments/api/requests.create
+
+The following parameters **must** be provided with the request:
+
+| Parameter  | Description                                 |
+|------------|---------------------------------------------|
+| amount     | The payment amount in cents                 |
+| asset      | The currency - NZD or AUD                   |
+| merchantId | The ID of the merchant creating the request |
+
+<br>
+
+The following parameters can **optionally** be provided with the request:
+
+| Parameter         | Description                                                  |
+|-------------------|--------------------------------------------------------------|
+| clientId          | The ID of the merchant specific client configuration         |
+| description       | Description of the payment                                   |
+| externalReference | Unique merchant reference for the payment request            |
+| notifyUrl         | The URL that will receive **POST** requests from the webhook |
+
+An example URL that a **POST** request could be sent to would be the following:
+
+```
+https://service.centrapay.com/payments/api/requests.create?amount=100&asset=NZD&description=Test%20purchase&merchantId=1399b053-b3dd-4c5b-9859-b5bf5c2ac477&notifyUrl=https%3A%2F%2Fexample.com%2Fservice.notify
+```
+
+<br>
+
+### [Paying a payment request - Pocket Voucher](https://service.centrapay.com/payments/api/documentation#/requests/postRequestspay)
+
+#### Step 1. Test Voucher Allocation
+
+To generate a test value voucher text ‘CENTRALBONUS’ 393. <br>
+You will then receive a response text containing an 8 digit voucher code that has $20 of loaded credit. 
+
+>Note:  
+The received code is only valid for two weeks from the issue date.<br>
+You might get charged your standard text rates from your provider.
+
+
+#### Step 2. Redeem Test Voucher to Pay Request
+
+Voucher redemption is accomplished via sending a **POST** request to the following endpoint:<br>
+https://service.centrapay.com/payments/api/requests.pay
+
+The following parameters **must** be provided with the request query:
+
+| Parameter     | Description                                                               |
+|---------------|---------------------------------------------------------------------------|
+| authorization | The voucher code                                                          |
+| ledger        | The ledger to use for payment in this case `g.pocketvouchers.pv`          |
+| requestId     | The payment requestId that is generated when `/requests.create` is called |
+
+An example URL that a **POST** request could be sent to would be the following:
+
+```
+https://service.centrapay.com/payments/api/requests.pay?amount=100&authorization=09626714&ledger=g.pocketvouchers.pv
+```
+
+<br>
+
+### [Paying a payment request - QR code](https://service.centrapay.com/payments/api/documentation#/requests/postRequestspay)
```

Comment:
This section can be removed and we explain how to generate the QR code in the payment flow up above